css/src/index.css

```diff
@@ -155,9 +155,19 @@ dt {
 .sibling\:bg-grey-95 ~ * { @apply .bg-grey-95; }
 .sibling\:text-grey-50 ~ * { @apply .text-grey-50; }
 
+/* Search Bar Styles */
+*:focus ~ .sibling\:focus\:text-grey-50 { @apply .text-grey-50; }
```

Comment:
This is kind of the opposite of our `sibling` selector usage so far. In other places we put the class on the focus target, not the styles target (i.e. `.focus\:sibling\:style:focus ~ *` vs `:focus ~ .sibling\:focus\:style`).

Not that I don't think this approach makes more sense, at least in some scenarios, but it's confusing because now we're being inconsistent. The two also aren't the same. The first example says "whenever I'm in focus, give any of my siblings after me these styles" whereas the latter say "whenever any of my siblings before me are in focus, give me these styles."

And maybe you're already doing that with your ordering? `.sibling:focus:style` vs `.focus:sibling:style`? That's not the clearest, but it works.